src/Yahoo/Provider.php

```diff
@@ -23,6 +23,9 @@ class Provider extends AbstractProvider
      */
     protected function getAuthUrl($state)
     {
+        $this->scopes(['openid2']);
```

Comment:
It is not needed. Just set in the scopes class porperty.
Please see \SocialiteProviders\Zendesk\Provider::$scopes